EIPS/eip-2135.md

```diff
@@ -1,80 +1,103 @@
 ---
 eip: 2135
-title: Consumable Interface
+title: NFT Consumables
```

Comment:
Bikeshedding the title a bit. The title should include a bit more context about what "consume" means.

```suggestion
title: One Time Use NFTs
```

```suggestion
title: Limited Use NFTs
```

Reply:
Thanks for suggestion. Finding a good name is hard! 
For now I'd update the name `Consumable NFT for Ticketing`. Will look forward to community to give feedback